bin/runtime/src/lib.rs

```diff
@@ -655,21 +742,23 @@ impl_runtime_apis! {
     }
 
     impl primitives::AlephSessionApi<Block> for Runtime {
-        fn next_session_authorities() -> Result<Vec<AlephId>, AlephApiError> {
-            Aleph::next_session_authorities()
-        }
-
         fn authorities() -> Vec<AlephId> {
             Aleph::authorities()
         }
 
-        fn session_period() -> u32 {
-            Aleph::session_period()
+        fn millisecs_per_block() -> u64 {
+            Elections::millisecs_per_block()
```

Comment:
Why did we move it from Aleph to Elections? Not saying it is wrong, but just would like to know...

Reply:
It's more natural for `session_period` and `sessions_per_era` to be in pallet elections, so `millisecs_per_block` kind of joined them on the way but there is also a possibility that this parameter will be needed in connection to eras.